### ISSUE TITLE
processing of cryptography.fernet.InvalidToken exception

### ISSUE THREAD

**Issue by imbolc** [aio-libs/aiohttp_session#27]
For now get_session just cause this exception, and I don't understand what to do with this. I would prefer lib to create new session silently.

**Comment by asvetlov** [comment]
So you got exception in case of broken data?

**Comment by imbolc** [comment]
I got it in case of changing of session key.

**Comment by minamorl** [comment]
Hi, same issue occured in my environment. Traceback:
```
Traceback (most recent call last):
File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/cryptography/fernet.py", line 99, in decrypt
    h.verify(data[-32:])
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/cryptography/hazmat/primitives/hmac.py", line 69, in verify
    ctx.verify(signature)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/cryptography/hazmat/backends/openssl/hmac.py", line 81, in verify
    raise InvalidSignature("Signature did not match digest.")
cryptography.exceptions.InvalidSignature: Signature did not match digest.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/aiohttp/server.py", line 266, in start
    yield from self.handle_request(message, payload)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/aiohttp/web.py", line 90, in handle_request
    resp = yield from handler(request)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/aiohttp_session/__init__.py", line 129, in middleware
    response = yield from handler(request)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/mead/objects.py", line 94, in wrapped
    context = await create_context(request)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/mead/objects.py", line 35, in create_context
    session = await aiohttp_session.get_session(request)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/aiohttp_session/__init__.py", line 107, in get_session
    session = yield from storage.load_session(request)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/asyncio/coroutines.py", line 200, in coro
    res = func(*args, **kw)
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/aiohttp_session/cookie_storage.py", line 34, in load_session
    self._fernet.decrypt(cookie.encode('utf-8')).decode('utf-8')
  File "/home/minamorl/.pyenv/versions/3.5.0/lib/python3.5/site-packages/cryptography/fernet.py", line 101, in decrypt
    raise InvalidToken
cryptography.fernet.InvalidToken
```

**Comment by asvetlov** [comment]
Fixed by #29
